src/app/query/stacks/balance/stacks-ft-balances.hooks.ts

```diff
@@ -90,3 +93,41 @@ export function useTransferableStacksFungibleTokenAssetBalances(
     [assetBalances]
   );
 }
+
+/**
+ * @see https://github.com/leather-wallet/issues/issues/16
+ */
+type StacksFtTokensFilter = 'all' | 'supported' | 'non-supported';
```

Comment:
I think it might be better to be consistent with naming of filters and the components? Here, it would be better to use `unsupported` for naming, and the component (instead of using `...Unknown...` it could be `StacksUnsupportedTokenAssetList`?